aten/src/ATen/native/quantized/cuda/make_per_channel_quantized_tensor.cu

```diff
@@ -0,0 +1,36 @@
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cuda/Loops.cuh>
+
+namespace at {
+namespace native {
+
+Tensor make_per_channel_quantized_tensor_cuda(
```

Comment:
what does this function do?  Is this supposed to be different than quantizing a tensor per channel?  Does this create an empty tensor?

Reply:
its just a cuda analog for _make_per_channel_quantized_tensor_cpu in native_functions.yaml which is used in a number of tests. I'm basing it off of make_per_tensor_quantized_tensor_cuda.